### ISSUE TITLE
Can't find the 'test.txt' and 'new_train.txt' in function load_data

### ISSUE THREAD

**Issue by chenovopride** [scpei/REA#2]
hi, thanks for your work. I've been inspired a lot!

But I can't find these two files`'test.txt'` and `'new_train.txt'` in function `load_data`. May I ask whether these two files represent aligned entities, i.e. (ei, ej)? Is this the training set and test set that you created from `ent_iLLs` in the `zh_en` dataset? 

I am asking you because I did not see a question about this. Thank you and your time for your answer!

**Comment by LonelyRanger001** [comment]
I‘m trying to reproduce the results and meet the same problem. Have you solved this?
